src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -30,69 +34,181 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
+            return new Date(d.setDate(diff + i)); // i = 0 - monday
+        }
+
         $scope.updateSlots = function () {
             var slots = [];
             for (var i = 0; i < $scope.slotsForWeek.length; i++) {
                 for (var j = 0; j < $scope.slotsForWeek[i].length; j++) {
                     if ($scope.slotsForWeek[i][j].type !== AvailabilityEnum.empty.id) {
                         var slot = {
-                            slotsDate: getDayOfTheWeek(new Date(), j),
-                            person: {id: id},
+                            slotsDate: getDayOfTheWeek(new Date(), j + $scope.relativeDayNumber),
+                            person: {id:id}, // null?
                             slot: {id: i + 1},
                             type: {id: $scope.slotsForWeek[i][j].type}
                         };
                         slots.push(slot);
                     }
                 }
             }
+
             var startDate = $filter('date')(getDayOfTheWeek(new Date(), $scope.relativeDayNumber), "dd-MM-yyyy");
             var endDate = $filter('date')(getDayOfTheWeek(new Date(), 4 + $scope.relativeDayNumber), "dd-MM-yyyy");
-            tdprSlotsService.updateSlots(slots, id, startDate, endDate).then(function () {
-                $scope.showSubmitSuccess = true;
-                $timeout(function () {
-                    $scope.showSubmitSuccess = false;
-                }, 2000);
-            });
-        };
 
-        $scope.showPreviousWeek = function () {
-            $scope.relativeDayNumber -= 7;
-            $scope.clearTable();
-            $scope.getSlots(id);
-        };
+            tdprSlotsService.updateSlots(slots, id, startDate, endDate).then(function(){
+                            $scope.showSubmitSuccess = true;
+                            $timeout(function(){$scope.showSubmitSuccess=false;}, 2000);
+                        });
 
-        $scope.showNextWeek = function () {
-            $scope.relativeDayNumber += 7;
-            $scope.clearTable();
-            $scope.getSlots(id);
-        };
+            var note = {
+                         "id": $scope.noteContent.id,
+                         "person": {
+                           "id": id
+                         },
+                         "description": $scope.temporaryContent,
+                         "date": $filter('date')(getDayOfTheWeek(new Date(), $scope.relativeDayNumber), "yyyy-MM-dd")
+             }
+
+            sendNote(note);
+            $scope.hasSlotChanged = false;
+            $scope.hasNoteChanged = false;
+         };
+
+        function enableNoteEditing() {
+            $scope.hasNoteChanged = true;
+            $scope.buttonTitle = "Discard";
+            $scope.textAreaStyle = {'background-color':'white'};
+            $scope.editNote = true;
+        }
+
+        function disableNoteEditing() {
+            $scope.hasNoteChanged = false;
+            $scope.buttonTitle = "Edit note";
```

Comment:
If you use the content property in css you can change this value by changing class